### ISSUE TITLE
Gmusic don't switches tracks.

### ISSUE THREAD

**Issue by tort-dla-psa** [mopidy/mopidy-gmusic#183]
When i select a track to play from playlist which was created by manually loading a playlist or an album from google music, it works fine. But when i try to switch track, mopidy starts the same track from the beginning. To make a switch i have to select track twice and rapidely.

Properties was not changed since [this issue](https://github.com/mopidy/mopidy-gmusic/issues/182)

**Comment by SuperPrower** [comment]
I have exactly same issue. Let me provide a little bit more details.

I am using Mopidy 2.1.0, mopidy-gmusic-git from Arch User Repositories, with ncmpcpp. 

Everything works mostly fine, until I try to switch track manually (by pressing >). When I do this, everything on display have changed, including track name, artist, and length, but same song that I switched from keeps playing. It keeps playing in whatever direction and despite how many times I change track. Only solution is quickly change track back and forward. So, to switch track to the next one, I have to quickly tap `>, <, >` (for example).

**Comment by leachim6** [comment]
I am having this same issue on Fedora 27:
Installed from pip:
Mopidy==2.1.0
Mopidy-GMusic==2.0.0

**Comment by lieuwex** [comment]
FWIW: The URLs that mopidy-gmusic provides through `translate_uri` seems to be correct, when I play them in a browser or whatever it plays the correct song.

**Comment by lieuwex** [comment]
I also have this problem with mopidy-beets, so more reason to think this is an issue in mopidy's core.

**Comment by morgoth** [comment]
I have the same problem with music played from google. When I listen to music from my local disk (using "Mopidy-File") it switches fine.
It started happening around beginning of May for me

**Comment by expl0ratory** [comment]
I have the same problem, but I can't switch tracks at all.  I'm stuck on the first track I start until I kill mopidy and restart.

**Comment by insanum** [comment]
So yeah, this is an annoying issue. I started mopidy in the foreground with -v to study the debug logs. When this issue occurs the following log is always seen:
```
DEBUG    2018-06-08 16:31:17,024 [63466:MainThread] mopidy.audio.actor
  Race condition happened. See #1222 and #1430.
```
This refers to the following issues in mopidy [#1430](https://github.com/mopidy/mopidy/issues/1430) and [#1222](https://github.com/mopidy/mopidy/issues/1222).

It seems the problem doesn't occur if your well into a song. Maybe because it's fully buffered?

In any case, I've found using mpc on the command line as follows works around the issue. Note the `playlist` call is used as an example to get a track number in the current loaded queue.
```
% mpc playlist | cat -n
% mpc -q seek 100%; mpc -q play 13; sleep 1; mpc
% mpc -q seek 100%; mpc -q next; sleep 1; mpc
```
